packages/utils/package.json

```diff
@@ -27,7 +27,7 @@
     "build:watch": "yarn run build --watch",
     "build:release": "yarn clean && yarn build",
     "check-build": "node -e \"(async function() { await import('./lib/index.js') })()\"",
-    "check-types": "tsc && vitest --run --typecheck --dir test/types/",
```

Comment:
@nflaig somehow this vitest command scans through the new `test/perf` folder and it caused error, even after I added `--exclude` option. Note that the benchmarks go with mocha

I guess it's still ok to drop this check because we also scan through unit test

Reply:
We need to run type checks separately because of `--typecheck` flag and those are not checked as part of unit tests. This seems to be an issue on vitest side, it shouldn't even scan files in this folder, I tried all possible options from [here](https://vitest.dev/config/#typecheck) but none worked other than setting `ignoreSourceErrors=true` which I think is ok here and only affect typecheck tests.